docs/architecture/cloud-native/authentication-authorization.md

```diff
@@ -19,7 +19,7 @@ Many organizations still rely on local authentication services like Active Direc
 
 - [Authentication basics](https://docs.microsoft.com/azure/active-directory/develop/authentication-scenarios)
 - [Access tokens and claims](https://docs.microsoft.com/azure/active-directory/develop/access-tokens)
-- [It may be time to ditch your on premises authentication services](https://oxfordcomputergroup.com/resources/o365-security-native-cloud-authentication/)
+- [It may be time to ditch your on premises authentication services](https://oxfordcomputergroup.com/resources/o365-security-native-cloud-authentication)
```

Comment:
I don't understand this change. This new link redirects to https://oxfordcomputergroup.com/resources/o365-security-native-cloud-authentication/. I think the broken link you had here, I've already fixed it: https://github.com/dotnet/docs/commit/f3983a8943238e9f95af7dd0dd8e8815e86a5701#diff-e536355adda102be6e21c5c0a7865605

```suggestion
- [It may be time to ditch your on premises authentication services](https://oxfordcomputergroup.com/resources/o365-security-native-cloud-authentication/)
```

Reply:
Maira... Pandoc, the PDF generator throws a 404 on that link. Tried it both with and without the trailing slash. Removed it so that we could generate the PDF.  Please advise.

Reply:
The link before was missing a slash in the middle of the URL. Did you try to generate with the change I made or was it prior to that?